ably.d.ts

```diff
@@ -3236,6 +3240,14 @@ declare namespace Types {
      * @param channelOptions - A {@link ChannelOptions} object containing the cipher.
      */
     fromEncodedArray: (JsonArray: any[], channelOptions?: ChannelOptions) => PresenceMessage[];
+
+    /**
+     * Initialises a `PresenceMessage` from a `PresenceMessage`-like object.
+     *
+     * @param values - The values to intialise the `PresenceMessage` from.
+     * @param stringifyAction - Whether to convert the `action` field from a number to a string.
```

Comment:
We should not be exposing things like a `stringifyAction` param in the public api. There is no reason a user of the lib should have to know or care that we use numerical encoding in our internal wire protocol, and this doc string is meaningless to such a user.

I'm not a fan of having the one method to do conceptually quite different actions controlled by that param anyway, so in https://github.com/ably/ably-js/pull/1923 I've split it into two different methods, `fromValues(values: Properties<PresenceMessage>): Message`  suitable for public use, and `fromWireProtocol(values: WireProtocolPresenceMessage): PresenceMessage` as an internal method.